formstackTheme.css

```diff
@@ -0,0 +1,35 @@
+.fsBody {
```

Comment:
Where does this get included?

Reply:
This needs to be copy and pasted into the "theme" section of formstack, but I don't want to rely on storing it in there because I don't think there's any versioning and if someone overwrites it and saves we'd have to redo the work.

Open to storing it somewhere else, perhaps I should explain that in the readme

Reply:
I've updated the readme